torch/serialization.py

```diff
@@ -165,10 +170,15 @@ def _mps_deserialize(obj, location):
     if location == 'mps':
         return obj.mps()
 
+def _meta_deserialize(obj, location):
+    if location == 'meta':
+        return torch.UntypedStorage(obj.nbytes(), device='meta')
```

Comment:
At least in some places (maybe not in eager torch) `storage.nbytes` is used meaningfully : https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/serialization/pickler.cpp#L307

Reply:
...and? The logic here is symmetric with cuda_deserialize when `_torch_load_uninitialized` is False